bigquery/src/main/scala/magnolify/bigquery/TableRowType.scala

```diff
@@ -265,16 +267,23 @@ object TableRowField {
 
 private object NumericConverter {
   // https://cloud.google.com/bigquery/docs/reference/standard-sql/data-types#numeric-type
-  private val MaxPrecision = 38
-  private val MaxScale = 9
+  private val NumericMaxPrecision = 38
+  private val NumericMaxScale = 9
 
   def toBigDecimal(v: Any): BigDecimal = BigDecimal(v.toString)
-  def fromBigDecimal(v: BigDecimal): Any = {
+  private def validate(name: String, v: BigDecimal, maxPrecision: Int, maxScale: Int): Unit = {
     require(
-      v.precision <= MaxPrecision,
-      s"Cannot encode BigDecimal $v: precision ${v.precision} > $MaxPrecision"
+      v.precision <= maxPrecision,
+      s"Cannot encode $name $v: precision ${v.precision} > $maxPrecision"
     )
-    require(v.scale <= MaxScale, s"Cannot encode BigDecimal $v: scale ${v.scale} > $MaxScale")
+    require(v.scale <= maxScale, s"Cannot encode $name $v: scale ${v.scale} > $maxScale")
+  }
+  def fromBigDecimal(v: BigDecimal): Any = {
+    validate("BigDecimal", v, NumericMaxPrecision, NumericMaxScale)
     v.toString()
   }
+  def fromBigNumeric(v: BigNumeric): Any = {
+    validate("BigNumeric", v.wkt, BigNumeric.MaxNumericPrecision, BigNumeric.MaxNumericScale)
```

Comment:
can we have consistent naming between `NumericMaxPrecision`/`MaxNumericPrecision`?